### PR TITLE
Removes all \red and \blue's

### PR DESCRIPTION
**Pull request by printer16**
Removes all \red's and \blues in favor of span classes. \red things that were bold were replaced with danger because it's warning but bold.
Putting this here to make sure I didn't miss any /span's or anything like that while I purge the blues.
TODO:

- [x] Remove the rest of the blue.

### CODE REVIEW

File: tools/SS13SmoothingCutter/Cutter.dm [hunk 1]
```diff
@@ -9,7 +9,7 @@
 	if(isfile(dmifile) && (copytext("[dmifile]",-4) == ".dmi"))
 		SliceNDice(dmifile)
 	else
-		world << "\red Bad DMI file '[dmifile]'"
+		world << "<span class='warning'>Bad DMI file '[dmifile]'</span>"
```
Comment (Lohikar) [inline tools/SS13SmoothingCutter/Cutter.dm]: Yeah, don't edit this. This doesn't have a stylesheet, spans won't work.

**Comment by printer16** [comment]
I'll be back.

File: code/game/gamemodes/malfunction/newmalf_ability_trees/tree_networking.dm [hunk 2]
```diff
@@ -172,7 +172,7 @@
 				log_ability_use(user, "advanced encryption hack (FAIL - title: [reporttitle])")
 				return
 			log_ability_use(user, "advanced encryption hack (SUCCESS - title: [reporttitle])")
-			world << "\red New [company_name] Update available at all communication consoles."
+			world << "<span class='warning'>New [company_name] Update available at all communication consoles.</span>"
```
Comment (skull132) [inline code/game/gamemodes/malfunction/newmalf_ability_trees/tree_networking.dm]: This befits alert.

File: code/game/mecha/mecha.dm [hunk 3]
```diff
@@ -1788,9 +1788,9 @@
 		var/obj/item/mecha_parts/mecha_equipment/tool/passenger/P = passengers[pname]
 		var/mob/occupant = P.occupant
 
-		user.visible_message("\red [user] begins opening the hatch on \the [P]...", "\red You begin opening the hatch on \the [P]...")
+		user.visible_message("<span class='warning'>[user] begins opening the hatch on \the [P]...</span>", "<span class='warning'>You begin opening the hatch on \the [P]...</span>")
```
Comment (skull132) [inline code/game/mecha/mecha.dm]: Odd that the latter isn't a `notice`. Befits both, really!

File: code/game/objects/items/weapons/material/twohanded.dm.orig [hunk 4]
```diff
@@ -159,18 +159,18 @@
 		if(istype(A,/obj/structure/window))
 			var/obj/structure/window/W = A
 			W.shatter()
-<<<<<<< HEAD:code/game/objects/items/weapons/material/twohanded.dm
+<<<<<<< HEAD:code/game/objects/items/weapons/material/twohanded.dm
```
Comment (skull132) [inline code/game/objects/items/weapons/material/twohanded.dm.orig]: What the fuck, does this file even exist.

File: code/modules/admin/verbs/randomverbs.dm [hunk 5]
```diff
@@ -311,9 +311,9 @@ proc/get_ghosts(var/notify = 0,var/what = 2, var/client/C = null)
 			if(g.antagHUD)
 				g.antagHUD = 0						// Disable it on those that have it enabled
 				g.has_enabled_antagHUD = 2				// We'll allow them to respawn
-				g << "\red <B>The Administrator has disabled AntagHUD </B>"
+				g << "<span class='warning'>The Administrator has disabled AntagHUD</span> "
```
Comment (skull132) [inline code/modules/admin/verbs/randomverbs.dm]: Rogue space at the end here. Should just make it into a period.

File: code/modules/awaymissions/zlevel.dm [hunk 6]
```diff
@@ -3,7 +3,7 @@ proc/createRandomZlevel()
 		return
 
 	var/list/potentialRandomZlevels = list()
-	admin_notice("\red \b Searching for away missions...", R_DEBUG)
+	admin_notice("<span class='warning'>\b Searching for away missions...</span>", R_DEBUG)
```
Comment (skull132) [inline code/modules/awaymissions/zlevel.dm]: Since we're already looking at these.

`\b` is the same as `<b>...</b>`. So this should actually be `<span class='danger'>` without the `\b`.

File: code/modules/hydroponics/trays/tray.dm [hunk 7]
```diff
@@ -440,7 +440,7 @@
 	weedlevel = 0
 
 	update_icon()
-	visible_message("<span class='danger'>The </span><span class='notice'>[previous_plant]</span><span class='danger'> has suddenly mutated into </span><span class='notice'>[seed.display_name]!</span>")
+	visible_message("<span class='danger'>The </span><span class='notice'>[previous_plant]</span><span class='danger'>has suddenly mutated into </span><span class='notice'>[seed.display_name]!</span>")
```
Comment (skull132) [inline code/modules/hydroponics/trays/tray.dm]: The space here is needed. Without it, it'd look like this:

`[previous_plant]has suddenly mutated into [seed.display_name]!`

File: code/modules/hydroponics/trays/tray_apiary.dm [hunk 8]
```diff
@@ -46,7 +46,7 @@
 /obj/machinery/apiary/attackby(var/obj/item/O as obj, var/mob/user as mob)
 	if(istype(O, /obj/item/queen_bee))
 		if(health > 0)
-			user << "\red There is already a queen in there."
+			user << "<span class='warning'> There is already a queen in there.</span>"
```
Comment (skull132) [inline code/modules/hydroponics/trays/tray_apiary.dm]: Rouge space after `<span>`.

File: code/modules/hydroponics/trays/tray_apiary.dm [hunk 9]
```diff
@@ -65,14 +65,14 @@
...
-			user << "\red <b>You begin to dislodge the apiary from the tray, the bees don't like that.</b>"
+			user << "<span class='danger'>You begin to dislodge the apiary from the tray, the bees don't like that.</span>"
 			angry_swarm(user)
 		else
 			user << "\blue You begin to dislodge the dead apiary from the tray."
 		if(do_after(user, 50))
 			new hydrotray_type(src.loc)
 			new /obj/item/apiary(src.loc)
-			user << "\red You dislodge the apiary from the tray."
+			user << "<span class='warning'> You dislodge the apiary from the tray.</span>"
```
Comment (skull132) [inline code/modules/hydroponics/trays/tray_apiary.dm]: Rouge space.

File: code/modules/hydroponics/trays/tray_apiary.dm [hunk 10]
```diff
@@ -86,7 +86,7 @@
 		var/obj/item/weapon/reagent_containers/glass/G = O
 		if(harvestable_honey > 0)
 			if(health > 0)
-				user << "\red You begin to harvest the honey. The bees don't seem to like it."
+				user << "<span class='warning'> You begin to harvest the honey. The bees don't seem to like it.</span>"
```
Comment (skull132) [inline code/modules/hydroponics/trays/tray_apiary.dm]: Spess.

File: code/modules/mining/machine_processing.dm [hunk 11]
```diff
@@ -34,7 +34,7 @@
 		return
 
 	if(!allowed(user))
-		user << "\red Access denied."
+		user << "<span class='warning'> Access denied.</span>"
```
Comment (skull132) [inline code/modules/mining/machine_processing.dm]: Spess.

File: code/modules/mining/mine_turfs.dm [hunk 12]
```diff
@@ -162,7 +162,7 @@
 /turf/simulated/mineral/attackby(obj/item/weapon/W as obj, mob/user as mob)
 
 	if (!usr.IsAdvancedToolUser())
-		usr << "\red You don't have the dexterity to do this!"
+		usr << "<span class='warning'> You don't have the dexterity to do this!</span>"
```
Comment (skull132) [inline code/modules/mining/mine_turfs.dm]: Spess.

File: code/modules/mining/satchel_ore_boxdm.dm [hunk 13]
```diff
@@ -72,7 +72,7 @@
 	set src in view(1)
 
 	if(!istype(usr, /mob/living/carbon/human)) //Only living, intelligent creatures with hands can empty ore boxes.
-		usr << "\red You are physically incapable of emptying the ore box."
+		usr << "<span class='warning'> You are physically incapable of emptying the ore box.</span>"
```
Comment (skull132) [inline code/modules/mining/satchel_ore_boxdm.dm]: Spess in this file too, along with line 88.

File: code/modules/mob/dead/observer/observer.dm [hunk 14]
```diff
@@ -289,11 +289,11 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 		return
 	var/aux_staff = check_rights(R_MOD|R_ADMIN, 0)
 	if(!config.antag_hud_allowed && (!client.holder || aux_staff))
-		src << "\red Admins have disabled this for this round."
+		src << "<span class='warning'>Admins have disabled this for this round.</span>"
 		return
 	var/mob/dead/observer/M = src
 	if(jobban_isbanned(M, "AntagHUD"))
-		src << "\red <B>You have been banned from using this feature</B>"
+		src << "<span class='danger'>You have been banned from using this feature"
```
Comment (skull132) [inline code/modules/mob/dead/observer/observer.dm]: Unclosed tag.

File: code/modules/mob/living/carbon/brain/life.dm [hunk 15]
```diff
@@ -23,9 +23,9 @@
 				if(prob(5))
 					apply_radiation(-5)
 					if(!container)
-						src << "\red You feel weak."
+						src << "<span class='warning'>You feel weak.</span>"
 					else
-						src << "\red STATUS: DANGEROUS LEVELS OF RADIATION DETECTED."
+						src << "<span class='warning'>STATUS: DANGEROUS LEVELS OF RADIATION DETECTED.</span>"
```
Comment (skull132) [inline code/modules/mob/living/carbon/brain/life.dm]: This sounds more like a `danger` to me.

File: code/modules/mob/living/carbon/carbon.dm [hunk 16]
```diff
@@ -211,7 +211,7 @@
 				if(!org.is_usable())
 					status += "dangling uselessly"
 				if(status.len)
-					src.show_message("My [org.name] is <span class='warning'> [english_list(status)].</span>",1)
+					src.show_message("My [org.name] is <span class='warning'>[english_list(status)].</span>",1)
 				else
 					src.show_message("My [org.name] is <span class='notice'> OK.</span>",1)
```
Comment (skull132) [inline code/modules/mob/living/carbon/carbon.dm]: There's an unnecessary space here as well. (Same as the above you fixed.)

File: code/modules/mob/living/carbon/human/emote.dm [hunk 17]
```diff
@@ -72,7 +72,7 @@
 
 			if (src.client)
 				if (client.prefs.muted & MUTE_IC)
-					src << "\red You cannot send IC messages (muted)."
+					src << "<span class='warning'>You cannot send IC messages (muted).</span>"
```
Comment (skull132) [inline code/modules/mob/living/carbon/human/emote.dm]: I think all muted messages should perhaps be `danger`? The one above this one was.

File: code/modules/mob/living/carbon/human/species/xenomorphs/alien_facehugger.dm [hunk 18]
```diff
@@ -48,11 +48,11 @@ var/const/MAX_ACTIVE_TIME = 400
 	..(user)
 	switch(stat)
 		if(DEAD,UNCONSCIOUS)
-			user << "\red \b [src] is not moving."
+			user << "<span class='warning'>\b [src] is not moving.</span>"
```
Comment (skull132) [inline code/modules/mob/living/carbon/human/species/xenomorphs/alien_facehugger.dm]: `\b` here as well. Swap to `danger`, remove `\b`.

No 4chin here.

File: code/modules/mob/living/simple_animal/borer/borer_captive.dm [hunk 19]
```diff
@@ -7,7 +7,7 @@
 
 	if (src.client)
 		if(client.prefs.muted & MUTE_IC)
-			src << "\red You cannot speak in IC (muted)."
+			src << "<span class='warning'>You cannot speak in IC (muted).</span>"
```
Comment (skull132) [inline code/modules/mob/living/simple_animal/borer/borer_captive.dm]: Muted message.

File: code/modules/mob/living/simple_animal/borer/say.dm [hunk 20]
```diff
@@ -14,7 +14,7 @@
 
 	if (src.client)
 		if(client.prefs.muted & MUTE_IC)
-			src << "\red You cannot speak in IC (muted)."
+			src << "<span class='warning'>You cannot speak in IC (muted).</span>"
```
Comment (skull132) [inline code/modules/mob/living/simple_animal/borer/say.dm]: Muted message.

File: code/modules/mob/living/simple_animal/hostile/syndicate.dm [hunk 21]
```diff
@@ -65,21 +65,21 @@
 			if (O.damtype == HALLOSS)
 				damage = 0
 			health -= damage
-			visible_message("\red \b [src] has been attacked with the [O] by [user]. ")
+			visible_message("<span class='warning'>\b [src] has been attacked with the [O] by [user].</span>")
```
Comment (skull132) [inline code/modules/mob/living/simple_animal/hostile/syndicate.dm]: 4chin reference again (`\b`)

File: code/modules/organs/blood.dm [hunk 22]
```diff
@@ -84,10 +84,10 @@ var/const/BLOOD_VOLUME_SURVIVE = 122
 					pale = 1
 					update_body()
 					var/word = pick("dizzy","woosey","faint")
-					src << "\red You feel [word]"
+					src << "<span class='warning'>ou feel [word]</span>"
```
Comment (skull132) [inline code/modules/organs/blood.dm]: Error in replacing: `ou` vs `you`.

File: code/modules/organs/blood.dm [hunk 23]
```diff
@@ -84,10 +84,10 @@ var/const/BLOOD_VOLUME_SURVIVE = 122
...
 				if(prob(1))
 					var/word = pick("dizzy","woosey","faint")
-					src << "\red You feel [word]"
+					src << "<span class='warning'>ou feel [word]</span>"
```
Comment (skull132) [inline code/modules/organs/blood.dm]: Here too.

File: code/modules/organs/organ_external.dm [hunk 24]
```diff
@@ -888,8 +888,8 @@ Note that amputating the affected organ does in fact remove the infection from t
 
 	if(owner)
 		owner.visible_message(\
-			"\red You hear a loud cracking sound coming from \the [owner].",\
-			"\red <b>Something feels like it shattered in your [name]!</b>",\
+			"<span class='warning'>You hear a loud cracking sound coming from \the [owner].",\
```
Comment (skull132) [inline code/modules/organs/organ_external.dm]: Unclosed tag.

File: code/modules/power/fractal_reactor.dm [hunk 25]
```diff
@@ -18,7 +18,7 @@
 /obj/machinery/power/fractal_reactor/New()
 	..()
 	if(!mapped_in)
-		world << "<b>\red WARNING: \black Map testing power source activated at: X:[src.loc.x] Y:[src.loc.y] Z:[src.loc.z]</b>"
+		world << "<b><span class='warning'>WARNING:</span> \black Map testing power source activated at: X:[src.loc.x] Y:[src.loc.y] Z:[src.loc.z]</b>"
```
Comment (skull132) [inline code/modules/power/fractal_reactor.dm]: Remove the `\black` here, as discussed.

File: code/modules/vehicles/cargo_train.dm [hunk 26]
```diff
@@ -162,8 +162,8 @@
 
 	if(is_train_head() && istype(load, /mob/living/carbon/human))
 		var/mob/living/carbon/human/D = load
-		D << "\red \b You ran over [H]!"
-		visible_message("<B>\red \The [src] ran over [H]!</B>")
+		D << "<span class='warning'>\b You ran over [H]!</span>"
```
Comment (skull132) [inline code/modules/vehicles/cargo_train.dm]: `\b` here. Swap to danger.

File: tools/SS13SmoothingCutter/Cutter.dm (hunk 1 above)
Comment (skull132) [inline tools/SS13SmoothingCutter/Cutter.dm]: What his nuts said.

File: code/modules/admin/verbs/randomverbs.dm [hunk 27]
```diff
@@ -62,7 +62,7 @@
 				M << "\bold You hear a voice in your head... \italic [msg]"
 
 	log_admin("SubtlePM: [key_name(usr)] -> [key_name(M)] : [msg]",admin_key=key_name(usr),ckey=key_name(M))
-	message_admins("\blue \bold SubtleMessage: [key_name_admin(usr)] -> [key_name_admin(M)] : [msg]", 1)
+	message_admins("<span class='notice'>\bold SubtleMessage: [key_name_admin(usr)] -> [key_name_admin(M)] : [msg]</span>", 1)
```
Comment (skull132) [inline code/modules/admin/verbs/randomverbs.dm]: `\bold` is present. Time to stab it!

**Comment by skull132** [comment]
```
3.59s$ python tools/TagMatcher/tag-matcher.py ../..
../../Aurorastation/Aurora.3/code/modules/admin/verbs/buildmode.dm
	<span> - Excess of 2 opening tag(s)
../../Aurorastation/Aurora.3/code/game/mecha/mecha.dm
	<span> - Excess of 1 opening tag(s)
../../Aurorastation/Aurora.3/code/modules/mob/living/silicon/robot/analyzer.dm
	<span> - Excess of 1 opening tag(s)
../../Aurorastation/Aurora.3/code/modules/admin/topic.dm
	<span> - Excess of 1 opening tag(s)
Total number of mismatches: 5
```

Travis is still reporting unmatched tags.

File: code/modules/admin/verbs/randomverbs.dm [hunk 28]
```diff
@@ -62,7 +62,7 @@
 				M << "\bold You hear a voice in your head... \italic [msg]"
 
 	log_admin("SubtlePM: [key_name(usr)] -> [key_name(M)] : [msg]",admin_key=key_name(usr),ckey=key_name(M))
-	message_admins("<span class='notice'>\bold SubtleMessage: [key_name_admin(usr)] -> [key_name_admin(M)] : [msg]</span>", 1)
+	message_admins("<span class='notice'>SubtleMessage: [key_name_admin(usr)] -> [key_name_admin(M)] : [msg]</span>", 1)
```
Comment (Lohikar) [inline code/modules/admin/verbs/randomverbs.dm]: Why did you remove this bold without adding a bold tag? Is this inherently bold?

**Comment by skull132** [comment]
Since this is awaiting resolution of Lohikar's comment. Can you also edit .travis.yml and change `MACRO_COUNT=1156` to `MACRO_COUNT=120`?
